src/menu/InputEditor.cpp

```diff
@@ -12,7 +12,7 @@ namespace Ship {
 #define SEPARATION() ImGui::Dummy(ImVec2(0, 5))
 
 void InputEditor::Init() {
-    BtnReading = -1;
+    mBtnReading = -1;
```

Comment:
i'd probably go for `mButton` instead of `mBtn` everywhere

Reply:
Out of scope.